src/app/component/eco-news/components/news-preview-page/news-preview-page.component.html

```diff
@@ -1,4 +1,8 @@
-<div class="container">
+<ng-template [ngIf]="isPosting">
+  <app-post-news-loader></app-post-news-loader>
+</ng-template>
```

Comment:
What for <code>[ngIf]</code> doesn't correspond to convention?

Reply:
fixed